### DISCUSSION TITLE
How to change control props on different screen size?

### DISCUSSION THREAD

**Discussion by mvskvortsov** [mantinedev/mantine#1643] [Q&A] [answered]
For example, I'm want to use SegmentedControl with orientation='horizontal' on table(>=768px) or desktop (>=1240px) screen sizes and orientation='vertical' on smaller displays (<768px).

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/theming/responsive/#changing-component-size-based-on-media-query

**Reply by mvskvortsov** [reply]
Oh, thx! I missed the option via useMediaQuery
